### PR TITLE
Restructure migrate how-to to fit new template

### PR DESCRIPTION
**Pull request by charislam**
# Description

Restructure migrate how-to to fit new template

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Partial #60

### CODE REVIEW

File: timescaledb/how-to-guides/migrate-data/index.md [hunk 1]
```diff
@@ -1,33 +1,15 @@
...
-<highlight type="note">
-When converting a normal SQL table to a hypertable, pay attention to how you
-handle constraints. A hypertable can contain foreign keys to normal SQL table
-columns, but the reverse is not allowed. UNIQUE and PRIMARY constraints must
-include the partitioning key.
-</highlight>
+# Data migration
+You can migrate your existing data into TimescaleDB in several ways.
 
+*   Migrate data when TimescaleDB is in the [same PostgreSQL instance][same-db]
```
Comment (Loquacity) [inline timescaledb/how-to-guides/migrate-data/index.md:4]: Don't forget to add in the "Learn about data migration ..." point at the top.

Comment (charislam) [inline timescaledb/how-to-guides/migrate-data/index.md:4]:
```suggestion
*   [Learn about data migration][about-data-migration] before you start
*   Migrate data when TimescaleDB is in the [same PostgreSQL instance][same-db]
```

File: timescaledb/how-to-guides/migrate-data/import-csv.md [hunk 2]
```diff
@@ -1,94 +1,64 @@
...
 
-In addition to parallelizing the workload, the tool also offers flags
-to improve the copy experience. [See the repo on GitHub][parallel importer] for full details.
+<highlight type="note">
+Timescale provides an open source [parallel
+importer](https://github.com/timescale/timescaledb-parallel-copy) program,
+`timescaledb-parallel-copy`, to speed up data copying. The program parallelizes
+migration by using several workers to run multiple `COPY`s concurrently. It also
+offers options to improve the copying experience. If you prefer not to download
+`timescaledb-parallel-copy`, you can also use regular PostgreSQL `COPY`.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/migrate-data/import-csv.md:21]:
```suggestion
Timescale provides an open source 
[parallel importer](https://github.com/timescale/timescaledb-parallel-copy) program,
`timescaledb-parallel-copy`, to speed up data copying. The program parallelizes
migration by using several workers to run multiple `COPY`s concurrently. It also
offers options to improve the copying experience. If you prefer not to download
`timescaledb-parallel-copy`, you can also use regular PostgreSQL `COPY`. 
```

File: timescaledb/how-to-guides/migrate-data/import-csv.md [hunk 3]
```diff
@@ -1,94 +1,64 @@
...
+    SELECT create_hypertable('<TABLE_NAME>', 'ts')
+    ```
+1.  At the command line, insert data into the hypertable from your `csv`. Use
+    `timescaledb-parallel-copy` to speed up migration. Adjust the number of
+    workers as desired. Alternatively see the next step.
+    ```bash
+    timescaledb-parallel-copy --db-name <DATABASE_NAME> --table <TABLE_NAME> \
+        --file <FILENAME>.csv --workers 4 --copy-options "CSV"
+    ``` 
+1.  **(Alternative)** If you don't want to use `timescaledb-parallel-copy`,
```
Comment (Loquacity) [inline timescaledb/how-to-guides/migrate-data/import-csv.md:47]:
```suggestion
1.  **OPTIONAL** If you don't want to use `timescaledb-parallel-copy`,
```

File: timescaledb/how-to-guides/migrate-data/same-db.md [hunk 4]
```diff
@@ -1,82 +1,75 @@
...
+    ```sql
+    SELECT create_hypertable('new_table', 'ts');
+    ```
+1.  Insert data from the old table to the new table.
+    ```sql
+    INSERT INTO new_table
+      SELECT * FROM old_table;
+    ```
+1.  If you created your new table without indexes in Step 1, recreate your
+    indexes.
```
Comment (Loquacity) [inline timescaledb/how-to-guides/migrate-data/same-db.md:62]:
```suggestion
1.  If you created your new table without indexes, recreate your indexes now.
```

Does this need a command?

File: timescaledb/how-to-guides/migrate-data/troubleshooting.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
+# Troubleshooting data migration problems
+
```
Comment (Loquacity) [inline timescaledb/how-to-guides/migrate-data/troubleshooting.md:2]: Can we add the boilerplate Troubleshooting stuff in here? You can see it in the caggs section.

**Review by Loquacity — CHANGES_REQUESTED** [review]

**Review by Loquacity — APPROVED** [review]

**Pull request merged**